bool block
float32 timeout
---
bool is_moving
